### ISSUE TITLE
NTR: Liver biopsy: BSEP protein decreased by immunohistochemical testing (Lab: pathology: biopsies & morphology)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#9261]
New term request Liver biopsy: BSEP protein decreased by immunohistochemical testing (elthat_100612144114): PFIC2, ABCB11; hepatic biopsy; bile salt exporter pump  UMLS: 4314027

**Comment by pnrobinson** [comment]
@MickeySegal

Adding new term: 
- ID: HP:6000718
- label: Decreased liver bile salt export pump expression
- parents: [Abnormal liver metabolite concentration (HP:0034644)](https://hpo.jax.org/app/browse/term/HP:0034644)
- pmid: PMID:22795478|PMID:20800306
- def: Amount of bile salt export pump (BSEP, coded by the ABCB11 gene) in liver tissue below the lower limit of normal.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/9261
- synonyms:
